erizo_controller/erizoClient/webpack.config.erizofc.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     libraryExport: 'default',
     library: 'Erizo',
-    libraryTarget: 'umd',
+    libraryTarget: 'var',
```

Comment:
erizofc should be umd, it's a different target. This is only used in node, mostly for Spine and umd is fine in this context

Reply:
ok, tought you'd prefer var